# Welcome To BrassicaEDB

* Introduction
* Our team
* Contributor
* How to cite
* Reference
* Acknowledgements

## Introduction

*Brassica* Expression Database (BrassicaEDB), a transcriptomic database of *Brassica* L. based on the Yu’s Triangle, we have collected six species of *Brassica* L. : three diploid basic species  *Brassica rapa* (AA, n = 10), *Brassica oleracea* (CC, n = 9) and *Brassica nigra* (BB, n = 8) , three tetraploid hybrid species *Brassica napus* (AACC, n = 19), *Brassica juncea* (AABB, n = 18), *Brassica carinata* (BBCC, n = 17), **2,992 transcriptome data** from **178 Bioprojects** in the NCBI Sequence Read Archive (SRA) database were analyzed, and the genomes of **39 different cultivars of six species of *Brassica* L.** were also collected. We used *Brassica carinata* 10167_V1, *Brassica juncea* Tumida_V2, *Brassica napus* ZS11, *Brassica nigra* NI100_V2, *Brassica oleracea* JZS_V2, *Brassica rapa* Chiifu_V4 were analyzed as reference genomes for each *Brassica* L.. The purpose of BrassicaEDB is to collect, organize, analyze, and display more transcriptome data of the six species of *Brassica* L., so that users can (1) query **detailed information** related to the genes of the six species; (2) To obtain the gene **expression levels** of six species under **different conditions, periods, tissues and treatments**; (3) Search for **transcription factors** and **regulatory networks** in six species; (4) Search the **gene co-expression network** of six species; (5) Search for **homologous genes** among six species; (6) To investigate the occurrence of **alternative splicing** in different species; (7) Use simple and convenient analytical tools.

## Our team

*Brassica*EDB is created by the Rapeseed Biology Innovation Team at Southwest University, led by Professor Lu Kun. Our team consists of dozens of outstanding postdoctoral researchers, PhD students, master's students, and undergraduates. Professor Lu Kun, as the head of the team, has been engaged in research on rapeseed genomics, molecular biology, and breeding for many years, with significant progress made in the areas of genetic improvement, yield enhancement, and quality improvement of rapeseed. Since its establishment, our team has been dedicated to improving the production capacity, quality, and stress resistance of rapeseed. The team has achieved remarkable accomplishments in areas such as rapeseed germplasm resources, genomics, and molecular marker technology. Our goal is to combine basic and applied research to provide scientific support for rapeseed production in China and promote the sustainable development of the rapeseed industry.

The main research directions of the team include:

**Genome Resource Mining and Big Data Analysis**: Our team utilizes high-throughput sequencing technology to conduct whole-genome analysis of rapeseed, exploring the genetic mechanisms behind important traits of rapeseed. We perform in-depth annotation of the rapeseed genome and, through big data analysis, identify genes related to desirable traits and genetic variations, providing data support for the improvement of rapeseed varieties.

**Application of Genome Editing Technology:** Our team actively promotes the application of gene editing technologies such as CRISPR-Cas9 in rapeseed, using precise gene editing techniques to enhance the yield and quality of rapeseed.

**Molecular Breeding and Functional Genomics**: Our team successfully identifies genes associated with key traits of rapeseed, such as stress resistance, yield, and oil content, through techniques like QTL analysis and association mapping. We have also utilized gene cloning technology to uncover the functions of these genes, providing molecular markers for breeding.

**Rapeseed Germplasm Innovation and Breeding**: Our team also focuses on the collection, preservation, and innovative use of rapeseed germplasm resources. Our conduct evaluation, utilization, and development of rapeseed germplasm to promote the sustainable use of rapeseed genetic resources.

**Digital Breeding Platform**: Our team has developed rapeseed bioinformatics platform, utilizing technologies such as big data analysis to drive the digitalization of rapeseed breeding.

## Contributor

Li xiaodong	(lxd1997xy@163.com)

Meng Boyu	(1340516624@qq.com)

Wang Yuhong	(wyhstar@email.swu.edu.cn)

Wang Xueying	(1500758738@qq.com)

## How to cite

Chao H, Li T, Luo C, Huang H, Ruan Y, Li X, Niu Y, Fan Y, Sun W, Zhang K, Li J, Qu C, Lu K. BrassicaEDB: A Gene Expression Database for *Brassica* Crops. *Int. J. Mol. Sci.* **2020**, *21*, 5831.



## Reference

[1] Xiaodong Li†, Boyu Meng†, Zhi Zhang, Lijuan Wei, Wei Chang, Yuhong Wang, Kai Zhang*, Tian Li*, Kun Lu*. qPrimerDB 2.0: an updated comprehensive gene-specific qPCR primer database for 1172 organisms. Nucleic Acids Research, 2024, gkae684. DOI:10.1093/nar/gkae684. 

[2] Chao H, Li T, Luo C, Huang H, Ruan Y, Li X, Niu Y, Fan Y, Sun W, Zhang K, Li J, Qu C, Lu K. BrassicaEDB: A Gene Expression Database for *Brassica* Crops. *Int. J. Mol. Sci.* **2020**, *21*, 5831.

[3] Jin, J., Tian, F., Yang, D.C., Meng, Y.Q., Kong, L., Luo, J. and Gao, G. (2017) [PlantTFDB 4.0: toward a central hub for transcription factors and regulatory interactions in plants](https://academic.oup.com/nar/article/45/D1/D1040/2290936). *Nucleic Acids Res*, **45**, D1040-D1045.

[4] Leonore Reiser, Erica Bakker, Sabarinath Subramaniam, Xingguo Chen, Swapnil Sawant, Kartik Khosa, Trilok Prithvi, Tanya Z Berardini, The Arabidopsis Information Resource in 2024, *Genetics*, Volume 227, Issue 1, May 2024, iyae027, https://doi.org/10.1093/genetics/iyae027

[5] Xiaoming Song, Yanping Wei, Dong Xiao, Ke Gong, Pengchuan Sun, Yiming Ren, Jiaqing Yuan, Tong Wu, Qihang Yang, Xinyu Li, Fulei Nie, Nan Li, Shuyan Feng, Qiaoying Pei, Tong Yu, Changwei Zhang, Tongkun Liu, Xiyin Wang, Jinghua Yang, *Brassica carinata* genome characterization clarifies U’s triangle model of evolution and polyploidy in *Brassica*, *Plant Physiology*, Volume 186, Issue 1, May 2021, Pages 388–406, https://doi.org/10.1093/plphys/kiab048

[6] *JBrowse 2: a modular genome browser with views of synteny and structural variation. Genome Biology (2023).* https://doi.org/10.1186/s13059-023-02914-z

## Acknowledgements

